internal/ignition/templates/discovery.ign

```diff
@@ -44,7 +44,7 @@
     "files": [{
       "overwrite": true,
       "path": "/usr/local/bin/agent-fix-bz1964591",
-      "mode": 755,
+      "mode": 493,
```

Comment:
As this is a Go template you may want to use `{{ 0755 }}`, that is easier to read and change, and translates into `493`.

Reply:
I was going for the smallest change that addressed the specific problem but agree there may be an issue with a broader scope that would address more consistent use of accepted best practices.